loco-new/base_template/src/controllers/home.rs

```diff
@@ -1,4 +1,5 @@
 use loco_rs::prelude::*;
+use axum::debug_handler;
```

Comment:
I don’t think this is related.
Since `axum::debug_handler` is now part of `loco_rs::prelude::*`; you can safely remove the import.